src/v2/Apps/Feature/Components/FeatureHeader/FeatureHeaderDefault.tsx

```diff
@@ -45,46 +42,47 @@ export interface FeatureHeaderDefaultProps {
 export const FeatureHeaderDefault: React.FC<FeatureHeaderDefaultProps> = ({
   feature: { name, subheadline, defaultImage: image },
 }) => {
-  return (
-    <Container
-      display={["block", "flex"]}
-      height={[
-        "auto",
-        !!image ? `calc(95vh - ${DESKTOP_NAV_BAR_HEIGHT}px)` : "50vh",
-      ]}
-    >
-      {image && (
-        <Figure height={["50vh", "auto"]}>
-          <picture>
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            <source srcSet={image.sm.srcSet} media="(max-width: 400px)" />
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            <source srcSet={image.md.srcSet} media="(max-width: 1200px)" />
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            <source srcSet={image.lg.srcSet} media="(min-width: 1200px)" />
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            <Image src={image.sm.src} alt={name} loading="lazy" />
-          </picture>
-        </Figure>
-      )}
+  if (image) {
+    return (
+      <FullBleed>
```

Comment:
Wondering if we could use https://github.com/artsy/force/blob/master/src/v2/Components/FullBleedHeader.tsx rather than this be a snowflake. The design is slightly different; just wondering if we want to consolidate for consistency. Would run it by design

Reply:
The design is indeed different, would love to reuse it but to make it I would require to change the `FullBleedHeader` implementation and the idea of it is quite different imo. If you think it's worthy abstracting more the `FullBleedHeader` to reuse here, I can try, but I guess the ideas of both situations are similar but different, we would be creating much more complexity on `FullBleedHeader` just to use here.

Example of feature page with FullBleed layout
![image](https://user-images.githubusercontent.com/15792853/135584934-724be86a-937b-45fd-9784-2a6f7de4d6e0.png)

Reply:
Rather than change `FullBleedHeader` my suggestion was that we should change the design to be more consistent with the patterns throughout the site. It'd be a question for design. Some of the header images would have to be updated, sure.

Reply:
Ah okay, now I got it. I will raise this flag to Jonathan, this image is one example of something that we would need to change, this feature sets freedom is a bit funky from what I see